expose `send` and `send_delayed` functions

zahimeen opened the pull request:
allow the user to make their own stuff using these functions.

zahimeen committed:
expose `send` and `send_delayed` functions

EvWilson commented:
Hey @zahimeen, looks great. Could we add these functions and some notes about them to the docs section in the README? Just want to make sure people are aware of them if we're making them available. Thanks!

zahimeen committed:
add `send` and `send_delayed` functions to docs

also limited the 'additional available functions' docs from going
beyond 80 characters to improve readability. furthermore, made the
example keymaps simpler.

EvWilson commented:
Thanks so much for your contribution, @zahimeen! Looks great, and is much appreciated.
